Description: Use new PHP 7.x style constructors.
Author: Mike Gabriel <mike.gabriel@das-netzwerkteam.de>

--- a/Horde_Form-2.0.20/lib/Horde/Form/Action.php
+++ b/Horde_Form-2.0.20/lib/Horde/Form/Action.php
@@ -27,7 +27,7 @@
     var $_params;
     var $_trigger = null;
 
-    function Horde_Form_Action($params = null)
+    function __construct($params = null)
     {
         $this->_params = $params;
         $this->_id = md5(mt_rand());
--- a/Horde_Form-2.0.20/lib/Horde/Form/Renderer.php
+++ b/Horde_Form-2.0.20/lib/Horde/Form/Renderer.php
@@ -60,7 +60,7 @@
      *                         parameter to Horde_Core_Ui_VarRenderer::factory().
      *                       - 'encode_title': @see $_encodeTitle
      */
-    function Horde_Form_Renderer($params = array())
+    function __construct($params = array())
     {
         global $registry;
         if (isset($registry) && is_a($registry, 'Registry')) {
--- a/Horde_Form-2.0.20/lib/Horde/Form/Variable.php
+++ b/Horde_Form-2.0.20/lib/Horde/Form/Variable.php
@@ -140,8 +140,8 @@
      * @param string $description    A long description of the variable's
      *                               purpose, special instructions, etc.
      */
-    function Horde_Form_Variable($humanName, $varName, $type, $required,
-                                 $readonly = false, $description = null)
+    function __construct($humanName, $varName, $type, $required,
+                         $readonly = false, $description = null)
     {
         $this->humanName   = $humanName;
         $this->varName     = $varName;
